manifests/profile/letsencrypt.pp

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Let's Encrypt System Requirements
+#
+# This installs certbot and sets it up with our contact email. It also
+# opens port 80 to the world, which is required for verifying ownership.
+class nebula::profile::letsencrypt (
+  String $email = lookup('nebula::profile::base::contact_email'),
+) {
+  class { 'letsencrypt':
+    email => $email,
+  }
+
+  firewall { '200 HTTP':
```

Comment:
Do we need to specify the chain in that protocol specific string format?

Reply:
You mean that `200 HTTP` title? It could be anything as long is it starts with something between 002 and 998.

Reply:
No, I was thinking of the `'INPUT:filter:IPv4'` stuff, but I think that is just for the chains, not the individual rules. All good!